api/src/database/migrations/20211004A-add-project-language.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from 'knex';
+import { getHelpers } from '../helpers';
+
+export async function up(knex: Knex): Promise<void> {
+	const helper = getHelpers(knex).schema;
+
+	await knex.schema.alterTable('directus_settings', (table) => {
+		table.string('project_language').notNullable().defaultTo('en-US');
+	});
+
+	if (helper.isOneOfClients(['cockroachdb', 'oracle'])) {
+		knex.schema.raw('ALTER TABLE directus_users ALTER COLUMN language SET DEFAULT NULL');
+	} else {
```

Comment:
Lets use the helpers for this, like:

https://github.com/directus/directus/blob/7d391ed4fad599240d0518fc2ab2c4bc9f20c7bf/api/src/database/migrations/20220303A-remove-default-project-color.ts#L7-L11